src/platforms/spark/TextTextureRendererSpark.mjs

```diff
@@ -0,0 +1,22 @@
+import TextTextureRenderer from "../../textures/TextTextureRenderer.mjs";
+
+export default class TextTextureRendererSpark extends TextTextureRenderer {
+    _draw() {
+        let textTextureRender = this;
+        let sparkText = sparkscene.create({ t: "text", text:textTextureRender._settings.text, pixelSize:textTextureRender._settings.fontSize*textTextureRender.getPrecision()});
+
+        let drawPromise = new Promise(function(resolve, reject) {
```

Comment:
Style: could just return new Promise
Style: prefer arrow function expression

Reply:
Now using the array syntax.